test/Calculations/test_Max.py

```diff
@@ -0,0 +1,30 @@
+from src.Calculations.Count import Count, get_calculation
+
+def test_get_calculation():
+    '''
+    Make sure the get_calculation function returns the calculation
+    '''
+    assert get_calculation() == Count
```

Comment:
make sure this test applies to this function (applies to all tests besides Count.py)